### DISCUSSION TITLE
Microsoft login option

### DISCUSSION THREAD

**Discussion by med-aymen-jelassi** [open-webui/open-webui#3784] [General]
As a user of Open web UI, I appreciate the current authentication methods available. However, it would be incredibly useful if you could integrate Microsoft login as an authentication option. Many organizations, including our team, rely heavily on Microsoft services, and having the ability to authenticate using Microsoft accounts would streamline our user experience and security practices.

Feature Details:

Benefit to Users: Simplified login process for users already using Microsoft services. Enhanced security and trust due to leveraging Microsoft's authentication protocols.
Integration Potential: Microsoft provides robust authentication APIs (such as Azure AD OAuth) that can be leveraged for seamless integration.
User Expectations: Users expect modern applications to offer multiple authentication options, including popular services like Microsoft.
Request:
Could the Open web UI team please consider adding Microsoft login as an authentication option? If this feature is already in the pipeline, could you kindly provide an estimated timeline for its release?

Additional Information:
This feature would not only improve convenience for existing users but also potentially attract new users who prefer Microsoft's authentication services. It aligns with current trends in software authentication practices and would enhance the overall usability of Ollama and lite llm.

Thank you for considering this request. I look forward to hearing about the team's plans regarding the addition of Microsoft login.

**Comment by tjbck** [comment]
https://docs.openwebui.com/getting-started/env-configuration#oauth

**Reply by roessletb** [reply]
This is close and we’re using it today, but we MUST get multi-tenant support. We’re a system and each institution lives in its own tenant.
